CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## securesystemslib v0.19.0
+
+### Added
+* Enable setting which GPG client to use through an environment variable (#315)
+
+### Changed
+* Dropped support for EOL Python 3.5 and 3.9 (#314)
```

Comment:
The way it's phrased here sounds like we are dropping support for python 3.9 :D

Reply:
Good catch! Let me fix that.